src/job/models/ingestion/seedingJobCreator.ts

```diff
@@ -125,7 +197,7 @@ export class SeedingJobCreator {
   private calculateGeometryByMode(mode: SeedMode, job: IngestionUpdateFinalizeJob | IngestionSwapUpdateFinalizeJob): Footprint | undefined {
     const logger = this.logger.child({ mode });
     logger.debug({ msg: 'Getting geometry for seeding job' });
-    if (mode === SeedMode.CLEAN) {
+    if (mode === SeedMode.CLEAN && job.type === this.config.get<string>('jobManagement.polling.jobs.swapUpdate.type')) {
```

Comment:
take it to const:
`const isSwapUpdate = job.type === this.config.get<string>('jobManagement.polling.jobs.swapUpdate.type')`
and reuse the `isSwapUpdate` within the if statement